ci-operator/config/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master.yaml

```diff
@@ -74,6 +79,20 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
+- as: precommit-check
+  commands: |
```

Comment:
@bshewale when I test in my machine, only works if:
`export GOFLAGS=`

not sure if works for you without that.